client/src/templates/Challenges/utils/frame.js

```diff
@@ -107,11 +113,17 @@ const initMainFrame = (frameReady, proxyLogger) => ctx => {
     // after the frame is ready. It has to be overwritten, as proxyLogger cannot
     // be added as part of createHeader.
     ctx.window.onerror = function(msg) {
-      console.log(msg);
+      var string = msg.toLowerCase();
+      var substring = 'script error';
+      if (string.indexOf(substring) > -1) {
```

Comment:
```suggestion
      if (string.includes('script error')) {
```

Reply:
'Suggestions cannot be applied on multi-line comments.' it says... Which makes basically no sense.

It's definitely cleaner, though, so I'll add them the hard way.

Reply:
@raisedadead I thought this was possible now.